INSTALL.md

```diff
@@ -68,28 +67,56 @@ pip install agent0[lateral] # Installs dependent sub-packages from git (e.g., et
 
 # Working with smart contracts
 
-We run tests and offer utilities that depend on executing bytecode compiled from Hyperdrive solidity contracts. This is not required to use elfpy.
+We run tests and offer utilities that depend on executing bytecode compiled from Hyperdrive solidity contracts.
+This is not required to use elfpy.
 
 NOTE: The Hyperdrive solidity implementation is currently under security review, and thus is not available publicly.
 The following instructions will not work for anyone who is not a member of Delv.
 
 ## 1. Set up smart contracts
 
-Clone the hyperdrive repo, then create a [sym link](https://en.wikipedia.org/wiki/Symbolic_link#POSIX_and_Unix-like_operating_systems) at `hyperdrive_solidity/` pointing to the repo location.
+Clone the hyperdrive repo:
 
 ```bash
 git clone git@github.com:delvtech/hyperdrive.git ../hyperdrive
-ln -s ../hyperdrive hyperdrive_solidity
 ```
 
 ## 2. Install Hyperdrive
 
 Complete the steps in Hyperdrive's [Pre-requisites](https://github.com/delvtech/hyperdrive#pre-requisites) and [Build](https://github.com/delvtech/hyperdrive#build) sections.
 
+## 2. Copy ABI & bytecode files
```

Comment:
This isn't strictly necessary for installing to a laymen right? We commit generated code and published ABIs. Maybe should go elsewhere instead of INSTALL.md

Reply:
Also numbering is off

Reply:
fixed numbering 👍 

this is in the `# Working with smart contracts` section, so it is separated from the layman install process.